index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
- [ ] WELL DONE 🎉. Your coding part is complete, however, there is one thing that needs to fix is your README. Please use[ this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project 🏆🍀